### PR TITLE
implimented language toggle button, with env vars

### PR DESCRIPTION
**Pull request by liamStockwellGDS**
[OJ-2424] NINO | FE | Language Translation Toggle

### What changed

updated Readme with ENV Vars information.
added new npm packages for the translation toggle.
added locale strings for the toggle
implemented the toggle element
Add the automated tests for the language toggle.
updated the template.yaml and the docker.compose with the new changes.

### Why did it change

So that a User can switch from English to Welsh anywhere in the journey

### Issue tracking

- [OJ-2424]https://govukverify.atlassian.net/browse/OJ-2424

### Environment variables or secrets

- [x] Documented in the [README](./blob/main/README.md)
- [x] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [x] Update [README](./blob/main/README.md) with any new instructions or tasks

[OJ-2424]: https://govukverify.atlassian.net/browse/OJ-2424?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[OJ-2424]: https://govukverify.atlassian.net/browse/OJ-2424?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -116,7 +119,8 @@ setGTM({
 });
 
 router.use(getGTM);
-router.use(getAssetPath);
+router.use(getLanguageToggle);
+//router.use(getAssetPath);
```
Comment (Caitlin-cooling) [inline src/app.js:123]: How come this is commented out?

File: test/browser/step_definitions/translation.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+// Steps for any background tasks we need to do pre tests
```
Comment (Caitlin-cooling) [inline test/browser/step_definitions/translation.js:1]: Not sure this comment is accurate?

Comment (liamStockwellGDS) [reply test/browser/step_definitions/translation.js]: true, this has been removed

File: src/app.js (hunk 1 above)
Comment (liamStockwellGDS) [reply src/app.js]: I think this was accidental. it has been fixed

**Review by Caitlin-cooling — APPROVED** [review]

**Pull request merged**